tests/test_mkieee_ChatGPT.F90

```diff
@@ -0,0 +1,60 @@
+! Test the mkieee and rdieee functions in NCEPLIBS-g2 written by ChatGPT using the prompt "write a fortran test program that tests an array of positive and negative values which includes a very large number and zero for the mkieee()/rdieee() functions of the NCEPLIBS-g2 library"
+!
+! Aidan Hartnett
+
+!program ieee_array_large_conversion_test
```

Comment:
Don't comment out the lines:

program ieee_array_large_conversion_test

and

end program ieee_array_large_conversion_test